### DISCUSSION TITLE
Invalid caractere

### DISCUSSION THREAD

**Discussion by E-Mello** [elementary/triage#234] [Q&A] [answered]
Hey, please help-me
When i click ( ctrl left + dot (.) ) is show me the character "e".
Because of this, i don't know adjust my code in any editor.

Yours sincerely,
Édio de Melo Pereira.

[Error Image](https://drive.google.com/file/d/1ul1yh-GfVjnwHNrtPbHF66UyUydMR0eI/view?usp=share_link)

**Comment by jeremypw** [comment]
I get the same thing on a new Horus install.  It seems that that key combination enters "Compose" mode - however "Compose" is disabled by default and `<Control>.` is not one of the listed possibilities in the Keyboard Setttings.   If you want to compose characters by typing two characters (e.g. typing é  by pressing `<Compose>` followed by `e` and `'` then you have to set your choice of Compose key in the Keyboard settings.  This does not stop `<Control>.` having the unwanted action however.  You can disable it by assigning a custom keyboard shortcut to it that does nothing (or whatever you want).

We probably need to look into what is capturing `<Control>.` by default and either make it work properly as a Compose key or disable it.

**Reply by E-Mello** [reply]
But would that be a problem with elementary itself? Or is it something I can fix? I didn't find something to adjust this

**Reply by ryonakano** [reply, marked as answer]
`Ctrl + .` is a shortcut key for IBus Emoji Picker and when you see the e character pressing Space would give you the picker. You can disable this shortcut key from ibus-setup:

![スクリーンショット 2023-05-10 07 38 32](https://github.com/elementary/triage/assets/26003928/0a719300-9478-4f9c-8595-2bee0ab66197)

**Reply by E-Mello** [reply]
Thank you very much, you saved me, and on top of that I learned that elementary has this IBus functionality to add emoji.

**Comment by jeremypw** [comment]
I learnt something too - thanks.  One question: why does the picker not open immediately? What other characters can be usefully entered after the `e`?

**Reply by Marukesu** [reply]
you can write a emoji name, like `smile`, or an written emoji, like `:)`, and then press space.
